source/_integrations/swiss_public_transport.markdown

```diff
@@ -41,6 +41,21 @@ name:
 
 The public timetables are coming from [Swiss public transport](https://transport.opendata.ch/).
 
-### Defining a custom polling interval
+## Actions
+
+The Swiss public transport integration has the following action:
+
+- `swiss_public_transport.fetch_connections`
+
+### Action `swiss_public_transport.fetch_connections`
+
+Fetch the connections for a specific instance.
+
+| Data attribute | Optional | Description                                              |
+|------------------------|----------|----------------------------------------------------------|
+| `config_entry_id`      | No       | The ID of the Swiss public transport config entry to get data from.|
+| `limit`                | Yes      | The number of connections to fetch. (default: 3, range: [1,15])|
```

Comment:
is this the number of connections after the specified time?

Reply:
What specified time?